src/plugins/performanceIndicator/plugin.js

```diff
@@ -58,5 +64,58 @@ export default function PerformanceIndicator() {
         indicator.statusClass('s-status-error');
       }
     }
+
+    function showOverlay() {
+      const overlayStylesText = `
+          #c-performance-indicator--overlay {
+            background-color:rgba(0,0,0,0.5);
+            position: absolute;
+            width: 300px;
+            left: calc(50% - 300px);
+          }
+      `;
+      const overlayMarkup = `
+        <div id="c-performance-indicator--overlay">
+          <table id="c-performance-indicator--table">
+            <tr class="c-performance-indicator--row"><td class="c-performance-indicator--measurement-name"></td><td class="c-performance-indicator--measurement-value"></td></tr>
+          </table>
+        </div>
+      `;
+      const overlayTemplate = document.createElement('div');
+      overlayTemplate.innerHTML = overlayMarkup;
+      const overlay = overlayTemplate.cloneNode(true);
+      overlay.querySelector('.c-performance-indicator--row').remove();
+      const overlayStyles = document.createElement('style');
+      overlayStyles.appendChild(document.createTextNode(overlayStylesText));
+
+      document.head.appendChild(overlayStyles);
+      document.body.appendChild(overlay);
+
+      indicator.off('click', showOverlay);
+
+      const interval = setInterval(() => {
+        overlay.querySelector('#c-performance-indicator--table').innerHTML = '';
+
+        for (const [name, value] of openmct.performance.measurements.entries()) {
+          const newRow = overlayTemplate
+            .querySelector('.c-performance-indicator--row')
+            .cloneNode(true);
+          newRow.querySelector('.c-performance-indicator--measurement-name').innerText = name;
+          newRow.querySelector('.c-performance-indicator--measurement-value').innerText = value;
+          overlay.querySelector('#c-performance-indicator--table').appendChild(newRow);
+        }
+      }, 1000);
+
+      indicator.on(
+        'click',
+        () => {
+          overlayStyles.remove();
+          overlay.remove();
+          indicator.on('click', showOverlay);
+          clearInterval(interval);
+        },
+        { once: true, capture: true }
+      );
+    }
```

Comment:
we can add a very simple test to add this to the default openmct, run, and get removed

Reply:
This is done.